notes.md

```diff
@@ -241,3 +241,9 @@ When playing back a track using the `TrackTimestampScale`, if the track is being
 It would be possible for a `Matroska Player` to also adjust the audio's samplerate at the same time as adjusting the timestamps if you wanted to play the two audio streams synchronously. It would also be possible to adjust the video to match the audio's speed. However, for playback, the selected track(s) timestamps SHOULD be adjusted if they need to be scaled.
 
 While the above example deals specifically with audio tracks, this element can be used to align video, audio, subtitles, or any other type of track contained in a Matroska file.
+
+## BlockAdditionMapping
+
+Extra data or metadata can be added to each frame independenly using `BlockAdditional` data. Each `BlockAdditional` is coupled with a `BlockAddID` that helps identify the kind of data it contains. The description of such data is done with the elements in `BlockAdditionMapping`.
+
+So far only the `BlockAddIDType` value of 0 is defined. It means the content of `BlockAdditional` is passed to the track decoder in addition of the frame data. It is up to the decoder to use the "complement" data or not and how to use them. If the `AlphaMode` flag is set, that means the extra data define an alpha layer for the video, for example in WebM it's a secondary VP8/VP9 stream.
```

Comment:
Per https://github.com/Matroska-Org/matroska-specification/blob/98c65bfccf38df4b1b2b7fdfeb9a0e2cba1ca979/ebml_matroska.xml#L164, the value `0` references the contents of the main block.

Also, IIUC, the codec mapping for [A_WAVPACK4](https://github.com/Matroska-Org/matroska-specification/blob/master/codec_specs.md#a_wavpack4), reserves the value of `1` for an id.